### ISSUE TITLE
Port the 2D Schrodinger to the framework

### ISSUE THREAD

**Issue by StillerPatrick** [Photon-AI-Research/NeuralSolvers#18]
- [ ] Split the EC Dataset into three datasets 
- [ ] Implement the Normalization Condition as a new designed  Boundary Condition (https://pytorch.org/docs/stable/generated/torch.trapz.html) could make things easier
- [ ] Integrate the new normalization condition into the PINN loss calculation
- [ ] Switch from x,y,t representation to a single tensor that represents all cases
- [ ] Integrate the implementation of the analytical solution in the IC dataset

**Comment by StillerPatrick** [comment]
-  [ ] Add pretraining on intital condition
